types/src/keyless/circuit_constants.rs

```diff
@@ -92,3 +97,19 @@ pub fn devnet_prepared_vk() -> PreparedVerifyingKey<Bn254> {
 
     PreparedVerifyingKey::from(vk)
 }
+
+pub struct Groth16TrapdoorSetup {
```

Comment:
Do you wanna move this into the simulation code, if this is a useful struct?

Reply:
not very useful beyond being a testing constant, that's why it was born here. can adjust in the future when the situation changes.